src/pages/post-test-summary/post-test-summary.ts

```diff
@@ -83,4 +111,13 @@ export class PostTestSummaryPage {
     });
     prompt.present();
   }
+
+  private mapSafetyQuestion(showMeQuestion?): string[] {
```

Comment:
Is the parameter really optional? Doesn't look like it from the function implementation

Reply:
good catch. It was at the beginning but recently it has changed

Reply:
ooo, sec. in constructor this method is executed without param (as at this stage we dont really know what was the show me quesiton)

Reply:
Ah okay